morpheus/_lib/src/io/deserializers.cpp

```diff
@@ -111,23 +111,23 @@ cudf::io::table_with_metadata load_table_from_file(const std::string& filename)
 int get_index_col_count(cudf::io::table_with_metadata& data_table)
 {
     int index_col_count = 0;
+    auto& col_names     = data_table.metadata.column_names;
 
     // Check if we have a first column with INT64 data type
-    if (data_table.metadata.schema_info.size() >= 1 &&
-        data_table.tbl->get_column(0).type().id() == cudf::type_id::INT64)
+    if (col_names.size() >= 1 && data_table.tbl->get_column(0).type().id() == cudf::type_id::INT64)
```

Comment:
We need a test that has integer columns in the input dataset to make sure we arent over zealous here